### PR TITLE
Xvisierra

### PR DESCRIPTION
**Pull request by xvisierra**
## Proposed changes

In this pull request, I have addressed the issue #9078 by adding example files to cs.csv as described in the issue ticket. I have also removed junk lines to improve the overall quality of the file. This change provides a solution to the issue by adding the requested file format examples to the documentation and cleaning up the data. I hope this change will be accepted by the maintainers.

## Checklist

- [x ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ x] I have added documentation to describe my feature.
- [x ] I have squashed my commits into logic units.
- [ x] I have described the changes in the commit messages.

## Other information

image

### CODE REVIEW

File: weblate/trans/tests/data/cs.csv [hunk 1]
```diff
@@ -1 +1,6 @@
-Thank you for using Weblate.,Děkujeme za použití Weblate.
+Key,Value
+"Hello, world!\n","Ahoj světe!\n"
+"Orangutan has %d banana.\n",""
+"Try Weblate at https://demo.weblate.org/!\n",""
+"Thank you for using Weblate.",""
+
```
Comment (nijel) [inline weblate/trans/tests/data/cs.csv:6]: This will most likely break tests.

File: weblate/trans/tests/data/cs.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
-  "Orangutan has %d banana.\n": "",
-  "Try Weblate at https://demo.weblate.org/!\n": "",
-  "Thank you for using Weblate.": ""
-}
+  {
+    "Hello, world!\n": "Ahoj světe!\n",
+    "Orangutan has %d banana.\n": "",
+    "Try Weblate at https://demo.weblate.org/!\n": "",
+    "Thank you for using Weblate.": ""
+  }
```
Comment (nijel) [inline weblate/trans/tests/data/cs.json:6]: What is reason for breaking formatting?

**Review by nijel — CHANGES_REQUESTED** [review]
This changes test files without a reason and does not add anything to the documentation, what is the actual topic of https://github.com/WeblateOrg/weblate/issues/9078.

**Comment by xvisierra** [comment]
> This changes test files without a reason and does not add anything to the documentation, what is the actual topic of #9078.

Ticket was there to add example files for the rest of the formats in the docu just like the json

**Comment by xvisierra** [comment]
> This changes test files without a reason and does not add anything to the documentation, what is the actual topic of #9078.

Ticket was there to add example files for the rest of the formats in the docu just like the json

**Pull request closed without merging**